src/visualization/components/internal/AxisTicksGenerator.tsx

```diff
@@ -111,20 +110,22 @@ const AxisTicksGenerator: FC<Props> = ({
         <Grid.Row className="tick-generator-input--custom">
           <ValueTickInput
             axisName={axisName}
+            tickPropertyName="totalTicks"
             tickOptions={generateAxisTicks}
             initialTickOptionValue={totalTicks}
             label="Total Tick Marks"
             update={update}
           />
           <TimeTickInput
             axisName={axisName}
+            tickPropertyName="tickStart"
             tickOptions={generateAxisTicks}
             initialTickOptionValue={tickStart}
-            label="Start Tick Marks At"
```

Comment:
how come this is removed?

Reply:
This prop wasn't being used. All of the `labels` in the child component have literal strings.